vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution.cc

```diff
@@ -232,7 +232,7 @@ ledger::PublisherInfoList Contribution::GetVerifiedListRecurring(
   }
 
   if (non_verified.size() > 0) {
-    ledger_->SaveUnverifiedContribution(std::move(non_verified));
+    ledger_->SaveUnverifiedContribution(std::move(non_verified), [](bool _){});
```

Comment:
Should this be using `base::DoNothing` instead?

Reply:
same as above